test/passenger_test.rb

```diff
@@ -68,7 +69,47 @@
     end
   end
 
+  #================
+  #total amount of money and total duration that passenger has spent
   describe "net_expenditures" do
```

Comment:
Include tests for the following edge cases: a driver with no trips and a driver with in progress trips.

It looks like these describe block includes tests for `net_expenditures` and `trip_duration`. That's ok! It'll be helpful for the readability of your test results to make the describe block title reflect this.